package.json

```diff
@@ -15,7 +15,7 @@
     "lint:html": "npm run vnu-jar",
     "regression": "ava test/tests",
     "regression-report": "node test/util/report",
-    "test": "npm run lint && npm run vnu-jar && npm run regression",
```

Comment:
hmmm `npm run lint` includes `npm run vnu-jar` so this is redundant, I believe!

Reply:
oh wow somehow I read this as the addition. The double html linting is removed in this PR :)